### PR TITLE
Fixed small android issue with firebase remote command, and added bra…

### PR DESCRIPTION
**Pull request by trister1997**
…ze remote command

### CODE REVIEW

File: remotecommands/tealium-react-braze/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { NativeModules } from 'react-native';
+
+const { TealiumReactBraze } = NativeModules;
+
+export default class BrazeRemoteCommand {
+    static name = "BrazeRemoteCommand";
+
+    static initialize() {
+        if (Platform.OS == 'ios') {
```
Comment (craigrouse) [inline remotecommands/tealium-react-braze/index.js:9]: @trister1997 I've probably missed something, but what's the reason this is only needed for iOS?

File: remotecommands/tealium-react-braze/ios/BrazeRemoteCommandWrapper.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
+//
+//  BrazeRemoteCOmmandWrapper.swift
```
Comment (craigrouse) [inline remotecommands/tealium-react-braze/ios/BrazeRemoteCommandWrapper.swift:2]: Very minor - extra capital in COmmand

File: remotecommands/tealium-react-braze/index.js (hunk 1 above)
Comment (trister1997) [reply remotecommands/tealium-react-braze/index.js:9]: Well on android, it the initialize function is an override from the parent it is inheriting from so it is called automatically, but needs to be called manually on iOS

**Review by jameskeith — APPROVED** [review]
couple of issues noted offline - otherwise good to go when addressed.

**Pull request merged**